src/lib/build.js

```diff
@@ -1,35 +1,23 @@
 const build = require('@netlify/build')
 
-const { getSiteInformation } = require('../utils/dev')
-
-const getBuildEnv = async ({ context }) => {
-  const { warn, error, netlify } = context
-  const { site, api, siteInfo } = netlify
-  const { teamEnv, addonsEnv, siteEnv } = await getSiteInformation({
-    api,
-    site,
-    warn,
-    error,
-    siteInfo,
-  })
-  const env = { ...teamEnv, ...addonsEnv, ...siteEnv }
-  return env
-}
-
 // We have already resolved the configuration using `@netlify/config`
 // This is stored as `netlify.cachedConfig` and can be passed to
 // `@netlify/build --cachedConfig`.
-const getBuildOptions = async ({ context, token, flags }) => {
-  const cachedConfig = JSON.stringify(context.netlify.cachedConfig)
+const getBuildOptions = ({ context, token, flags }) => {
+  const { cachedConfig } = context.netlify
+  const serializedConfig = JSON.stringify(cachedConfig)
   const { dry, debug } = flags
   // buffer = true will not stream output
   const buffer = flags.json || flags.silent
+  const env = Object.entries(cachedConfig.env).reduce(
```

Comment:
Do we need to pass `env` as a build option? Won't `@netlify/build` use the one in `cachedConfig`?
cc @ehmicky.

If we need it could we [sand] use:
```js
const env = fromEntries(Object.entries(cachedConfig.env).map(([key, variable]) => [key, variable.value]))
```

Reply:
I don't think we need it. @ehmicky's list had an item for removing it, which I somehow missed. I'll remove it. Thanks!